src/types/Tenant.ts

```diff
@@ -13,6 +13,10 @@ export default interface Tenant extends CreatedUpdatedProps {
       active: boolean;
       type: string;
     };
+    oicp?: {
+      active: boolean;
+      type: string;
```

Comment:
I don't think that type is necessary here has we don't have different implementations for Hubject

Reply:
Yes, I thought maybe in the future. OICP could also become a standard.